vktrace/vktrace_viewer/vktraceviewer_qtracefileloader.cpp

```diff
@@ -217,7 +218,18 @@ bool vktraceviewer_QTraceFileLoader::populate_trace_file_info(vktraceviewer_trac
         pTraceFileInfo->packetCount++;
         fileOffset += packetSize;
 
-        fseek(pTraceFileInfo->pFile, fileOffset, SEEK_SET);
+        seekResult = fseek(pTraceFileInfo->pFile, packetSize - sizeof(uint64_t), SEEK_CUR);
```

Comment:
Ah yes, this change was to fix opening really large files. Instead of seeking from the start of the file every time, this change will just seek relative to the current position, so we advance the pointer by the size of the packet minus the sizeof(uint64_t) which holds packetSize (which we've already read in).